### ISSUE TITLE
Multiple schemas for response code

### ISSUE THREAD

**Issue by blade2005** [asteasolutions/zod-to-openapi#31]
Does zod-to-openapi support different schema's for different content types but the same response code?

Such as
```
200:
  application/json:
    schema:
      $ref: "#/components/jsonResponse"
  text/csv:
    schema:
      $ref: "#/components/csvResponse"
```

**Comment by AGalabov** [comment]
Hi @blade2005,

Sorry for the delayed response. I saw this and had the idea of implementing it but never had the time. So basically - no we do not support this right now.

What I am thinking of implementing is something like:
```ts
      registry.registerPath({
        method: 'get',
        path: '/',
        responses: {
          200: {
            "application/json": {
              schema: JsonSchema
              description: 'JSON response',
            },
            "text/csv": {
              schema: CsvSchema
              description: 'CSV response',
            },
          },
        },
      });
```
It would be closer to the OpenAPI format (with the direct nesting of media types). Hopefully we should be able to open a PR during the weekend or early next week.

**Comment by ryan-mars** [comment]
Can we include requests and responses as part of this issue?

**Comment by AGalabov** [comment]
This feature should now be available as part of the [v2.0.0 release](https://github.com/asteasolutions/zod-to-openapi/releases/tag/v2.0.0).
